frontend/components/UserSettings.vue

```diff
@@ -0,0 +1,976 @@
+                                </div>
+                            </template>
+                        </Dropdown>
+                    </div>
+                </div>
+                <div class="flex items-center pb-4">
+                    <div class="h-[1.6px] w-10 bg-calypso-400" />
+                    <div class="flex-grow-5 mx-4 text-[1.65rem] font-semibold">
+                        <T key-name="dashboard.user.settings.delete.log.out" />
+                    </div>
+                    <div class="h-[1.6px] flex-grow bg-calypso-400" />
+                </div>
+                <div class="flex pb-5 pl-10">
+                    <div class="flex w-full flex-col">
+                        <div class="text-2xl">
+                            <T key-name="dashboard.user.settings.log.out" />
+                        </div>
+                        <div class="text-text dark:text-natural-300">
+                            <T
+                                key-name="dashboard.user.settings.log.out.description"
+                            />
+                        </div>
+                    </div>
+                    <div class="flex w-full items-center justify-end">
+                        <button
+                            class="w-44 rounded-md border-2 border-dandelion-300 bg-natural-50 py-0.5 text-base font-medium hover:bg-dandelion-200 dark:border-dandelion-300 dark:bg-natural-900 dark:hover:bg-pesto-600"
+                            @click="logoutUser"
+                        >
+                            <T key-name="dashboard.user.settings.log.out" />
+                        </button>
+                    </div>
+                </div>
+                <div
+                    class="flex pb-8 pl-10 text-mahagony-600 dark:text-mahagony-300"
+                >
+                    <div class="flex w-full flex-col">
+                        <div class="text-2xl">
+                            <T key-name="dashboard.user.settings.delete" />
+                        </div>
+                        <T
+                            key-name="dashboard.user.settings.delete.description"
+                        />
+                    </div>
+                    <div class="flex w-full items-center justify-end">
+                        <button
+                            class="w-44 rounded-md border-2 border-mahagony-500 bg-natural-50 py-0.5 text-base font-medium text-text hover:border-mahagony-600 hover:bg-mahagony-300 dark:bg-natural-900 dark:text-natural-50 dark:hover:bg-mahagony-500030"
+                            @click="
+                                isDeleteAccountDialogVisible =
+                                    !isDeleteAccountDialogVisible
+                            "
+                        >
+                            <T key-name="dashboard.user.settings.delete" />
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </Dialog>
```

Comment:
![image](https://github.com/user-attachments/assets/d46dd30c-1a50-425e-a101-6f91da744c80)
fix this arrow please